src/thunderbots/software/test/ai/hl/stp/evaluation/pass.cpp

```diff
@@ -0,0 +1,187 @@
+/**
+ * This file contains the unit tests for evaluation functions
+ * in pass.cpp
+ */
+
+#include "ai/hl/stp/evaluation/pass.h"
+
+#include <gtest/gtest.h>
+
+#include "test/test_util/test_util.h"
+
+TEST(PassEvaluationTest, findBestInterceptForBall_robot_on_ball_path)
```

Comment:
```suggestion
TEST(InterceptEvaluationTest, findBestInterceptForBall_robot_on_ball_path)
```
same goes for all tests

Reply:
changed all instances